commands/shell-server/cmd.go

```diff
@@ -0,0 +1,73 @@
+package shellserver
+
+import (
+	"fmt"
+	"net/http"
+	goruntime "runtime"
+	"time"
+
+	graceful "gopkg.in/tylerb/graceful.v1"
+
+	"github.com/taskcluster/taskcluster-worker/commands"
+	"github.com/taskcluster/taskcluster-worker/plugins/interactive"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+var defaultShell = "sh"
+
+func init() {
+	if goruntime.GOOS == "windows" {
+		defaultShell = "cmd.exe"
+	}
+	commands.Register("shell-server", cmd{})
+}
+
+type cmd struct{}
+
+func (cmd) Summary() string {
+	return "Host an interactive shell-server on localhost"
+}
+
+func (cmd) Usage() string {
+	return `
+taskcluster-worker shell-server will open a websocket to an interactive task, start
+a shell and expose it in your terminal. This is similar to using an SSH client.
```

Comment:
SSH is an abbreviation, and when pronouncing the letter "S" it sounds as if it's spelled "Ess". Hence, it's "an SSH" and not "a SSH"...

Okay, I could still be wrong, but in that case let's leave to for a naive English speaker  :) haha